backend/app/Services/WorkOrder/ChangeRequestService.php

```
@@ -0,0 +1,700 @@
+     * is the failure mode this whole workflow exists to prevent.
+     *
+     * @param  array<string, mixed>  $data
+     *
+     * @throws \DomainException
+     */
+    public function apply(WorkOrderChangeRequest $changeRequest, User $user, array $data = []): WorkOrderChangeRequest
+    {
+        $this->assertTransition($changeRequest, ChangeRequestStatus::Applied);
+
+        return DB::transaction(function () use ($changeRequest, $user, $data) {
+            $workOrder = WorkOrder::whereKey($changeRequest->work_order_id)->lockForUpdate()->first();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\WorkOrder::whereKey().


(staticMethod.notFound)